packages/document/docs/en/guide/advanced/custom-theme.mdx

```diff
@@ -62,6 +62,10 @@ const Layout = () => (
     beforeDoc={<div>beforeDoc</div>}
     /* Doc page end */
     afterDoc={<div>afterDoc</div>}
+    /* Doc content front */
+    beforeDoc={<div>beforeDoc</div>}
```

Comment:
small typo, it should be `beforeDocContent` and `afterDocContent` in docs

Reply:
Thanks! will fix it

Reply:
Fixed.